packages/next/src/build/webpack/plugins/flight-client-entry-plugin.ts

```diff
@@ -341,6 +345,15 @@ export class ClientReferenceEntryPlugin {
               'Server Actions require `experimental.serverActions` option to be enabled in your Next.js config: https://nextjs.org/docs/app/building-your-application/data-fetching/server-actions'
             )
           )
+        } else if (
+          this.serverActionsSizeLimit !== undefined &&
+          parseInt(this.serverActionsSizeLimit.toString()) < 1
+        ) {
+          compilation.errors.push(
+            new Error(
+              'Server Actions Size Limit must exceed 1 in number or filesize format to be enabled in your Next.js config: https://nextjs.org/docs/app/building-your-application/data-fetching/server-actions'
+            )
+          )
```

Comment:
These checks can be done when loading the config and don't need to be in the plugin for example, https://github.com/vercel/next.js/blob/c3c6214c09ab1fbcaaa7fe74e909f36eab93cbb7/packages/next/src/server/config.ts#L213-L217